### ISSUE TITLE
Some Pictures Uploaded to Busy.org are Cropped

### ISSUE THREAD

**Issue by DoctorLai** [busyorg/busy#1282]
I use busy.org to publish a post today and I noticed that one picture was cropped.

# How to Reproduce
Drag the photo to the editor.
![image.png](https://res.cloudinary.com/hpiynhbhq/image/upload/v1514327282/qcp4dpouijpjlc3gbajk.png)

after 100% upload process, the photo shown in the editor is cropped,
![image.png](https://res.cloudinary.com/hpiynhbhq/image/upload/v1514327321/ixwaxzd3fregfhn0dlhf.png)

When I publish the post, the photo is still cropped (so it is not a display issue in the editor).

I have to re-edit the post in steemit.com and re-upload the image using steemit editor, so here is the correct photo URL (by steemit):

```
https://steemitimages.com/DQmZ4n35NR4Ecu3uqzxdQAujXt6ALHmue5BEnMWXWpTENM8/2017-12-26%2012.19.04-1.jpg
```

and here is the version from busy.org

```
https://steemitimages.com/0x0/https://res.cloudinary.com/hpiynhbhq/image/upload/v1514327288/u7tzdhw0vi1dzkfspkiz.jpg
```

# Video that reproduces the bug
https://youtu.be/G1NaFSeCXoM

# Browser
- Chrome version 63
- Win 10 64 bit

<br /><hr/><em>Posted on <a href="https://utopian.io/utopian-io/@justyy/some-pictures-uploaded-to-busy-org-are-cropped">Utopian.io -  Rewarding Open Source Contributors</a></em><hr/>

**Comment by espoem** [comment]
the video as gif

![ezgif com-optimize 1](https://user-images.githubusercontent.com/16621517/34446344-81967db4-ecda-11e7-826c-96d043571d6e.gif)

**Comment by Sekhmet** [comment]
@bonustrack this is caused by proxying images via steemitimages. What do we do about it?

**Comment by Sekhmet** [comment]
It was fixed by #1654
